sympy/solvers/solveset.py

```diff
@@ -871,3 +872,295 @@ def solveset(f, symbol=None):
             return solveset_real(f, symbol)
         else:
             return solveset_complex(f, symbol)
+
+
+###############################################################################
+################################ LINSOLVE #####################################
+###############################################################################
+
+
+def linear_eq_to_matrix(equations, *symbols):
+    r"""
+    Converts a given System of Equations into Matrix form.
+    Here `equations` must be a linear system of equations in
+    `symbols`. The order of symbols in input `symbols` will
+    determine the order of coefficients in the returned
+    Matrix.
+
+    The Matrix form corresponds to the augmented matrix form.
+    For example:
+
+    4.x + 2.y + 3.z  = 1
```

Comment:
At some point, you may want re-do your docs with nice equations for sphinx, c.f. http://docs.sympy.org/latest/modules/solvers/ode.html#checkinfsol